tools/autograd/gen_annotated_fn_args.py

```diff
@@ -25,14 +25,17 @@
 from tools.codegen.model import BaseOperatorName, NativeFunction
 import tools.codegen.api.python as python
 from .gen_python_functions import should_generate_py_binding, is_py_torch_function, \
-    is_py_nn_function, is_py_linalg_function, is_py_variable_method
+    is_py_nn_function, is_py_linalg_function, is_py_variable_method, is_py_special_function, \
+    is_py_fft_function
 
 def gen_annotated(native_yaml_path: str, out: str, autograd_dir: str) -> None:
     native_functions = parse_native_yaml(native_yaml_path).native_functions
     mappings = (
         (is_py_torch_function, 'torch._C._VariableFunctions'),
         (is_py_nn_function, 'torch._C._nn'),
         (is_py_linalg_function, 'torch._C._linalg'),
+        (is_py_special_function, 'torch._C._special'),
+        (is_py_fft_function, 'torch._C._fft'),
```

Comment:
So far, added `special` functions had either just one argument (unary ufuncs) or expected `TensorLike()` args (`xlog1py`), so tests were passing (note: without this, the overrides test will test just `TensorLike()` inputs for the `nargs`), see:

https://github.com/pytorch/pytorch/blob/master/test/test_overrides.py#L600

```python
if func in annotated_args:
     # do stuff which special function should have done as well
     # Not True for special functions since args were not annotated yet
else:
    # ...
    nargs = len(args.args)
    if args.defaults is not None:
        nargs -= len(args.defaults)
    func_args = [instance_gen() for _ in range(nargs)]
```

Thanks to `@kshitij12345` for suggesting the quick fix! :) This has been fixed for `fft` functions as well, to prevent future failures.